### PR TITLE
First exercise

### PR DESCRIPTION
**Pull request by dhlavac**
Description

### CODE REVIEW

**Comment by Tiboris** [comment]
@dhlavac where's your description?

**Comment by dhlavac** [comment]
Added second exercise

File: xhlavacd/rebasing.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+index 87e4108..da02b91 100644
+--- a/xhlavacd/rebasing.md
++++ b/xhlavacd/rebasing.md
+@@ -1 +1,2 @@
+ Lorem ipsum dolor si amet!!!!
++For the Horde!!!
+ce1d63d65e000e6c87fe17a0600c4b4d462ea92c
+Lok'tar Ogar
+Sylvans is new warchief!!!!
+Aliance s..ks!!!
```
Comment (Hyperkid123) [inline xhlavacd/rebasing.md:13]: I don't think i can approve this message :dagger:

**Review by Hyperkid123 — APPROVED** [review]
Good job

File: xhlavacd/rebasing.md (hunk 1 above)
Comment (dhlavac) [reply xhlavacd/rebasing.md:13]: :smiley:

**Pull request closed without merging**